src/sections/Meshmap/Meshmap-design/meshmap-design-hero.js

```diff
@@ -92,8 +86,8 @@ const MeshmapHeroSection = () => {
         <p>MeshMap is the world’s only visual designer for Kubernetes and cloud native applications. Design, deploy, and manage your Kubernetes-based, cloud native deployments allowing you to speed up infrastructure configuration.</p>
       </div>
       <div className="hero-image">
-        <img className={imageInView ? "locator-moving" : "locator"} src={isDark ? MeshmapLocatorDark : MeshmapLocatorLight} alt="locator" />
-        <img className={imageInView ? "map map-visible" : "map"} src={isDark ? MeshmapImageBottomDark : MeshmapImageBottomLight} alt="integrations" ref={locatorRef} />
+        <img className={inView ? "locator-moving" : "locator"} src={isDark ? MeshmapLocatorDark : MeshmapLocatorLight} alt="locator" />
```

Comment:
can you explain this change?

Reply:
Here is small refactoring related to state management, based on this observation I have created another ticket refactor other pages: #5589

Reply:
Maybe I will remove this change as this is getting covered in #5604

Reply:
This factor is also impacting performance as state gets changed it will re render page and current implementation can lead to infinite loop in some of the edge cases